solvers/lasso_jl.jl

```diff
@@ -0,0 +1,36 @@
+using Distributions
+using GLM
+using Lasso
+using LinearAlgebra
+using PyCall
+using SparseArrays
```

Comment:
Do we need all these?
```suggestion
using Lasso
using PyCall
using SparseArrays
```